airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/test/pipeline.py

```diff
@@ -39,27 +55,105 @@ def get_test_steps(context: ConnectorContext) -> STEP_TREE:
         return []
 
 
-async def run_connector_test_pipeline(context: ConnectorContext, semaphore: anyio.Semaphore) -> ConnectorReport:
+def build_extra_params_mapping(
+    extra_params: Tuple[str], unique_steps_to_run: Dict[CONNECTOR_TEST_STEP_ID, Step]
```

Comment:
❓Is extra params a tuple or a list?

Reply:
It's a tuple of strings when received from the CLI, this function transforms it  to a dict